contracts/ReputationManager.sol

```diff
@@ -193,7 +193,7 @@ contract ReputationManager is Migratable, Ownable, Pausable {
         uint256 manufacturerReward;
         (irnReward, manufacturerReward) = getReputationRewards(msg.sender, _manufacturerWallet, _deviceId);
         _distributeRewards(_manufacturerWallet, msg.sender, irnReward);
-        _distributeRewards(_manufacturerWallet, _manufacturerWallet, manufacturerReward);
+        /*_distributeRewards(_manufacturerWallet, _manufacturerWallet, manufacturerReward);
```

Comment:
The commented code below is still needed isnt it?

Reply:
yes, removed comment